webapi/WebapiBuild.sbt

```diff
@@ -318,6 +319,8 @@ lazy val library =
 
         // Java EE modules which are deprecated in Java SE 9, 10 and will be removed in Java SE 11
         val jaxbApi                = "javax.xml.bind"                % "jaxb-api"                 % "2.2.12"
+
+        val icu4j                  = "com.ibm.icu"                   % "icu4j"                    % "62.1"
```

Comment:
Or do you want to leave `DateUtilV1 ` as it is? Could we make call v1 v2 methods?

Reply:
`DateUtilV1` uses jodd. I could refactor it to use the v2 code, but that would be more work. My idea was to avoid changing v1 if possible, because the priority is to finish v2.

Reply:
>  I could refactor it to use the v2 code, but that would be more work. My idea was to avoid changing v1 if possible, because the priority is to finish v2.

I understand. My concern is that both versions of the API have to behave identically when converting from and to JDN and that this  might be more difficult to assure if we use two different libraries for v1 and v2.

But of course, there could be a version update of one of those libraries and then we would have the same issue if the new version behaves differently from the old one.

Reply:
Ok, could you create a separate issue for that so we can do it later?

Reply:
#1007